### PR TITLE
Fixes to keypad docs

### PR DESCRIPTION
**Pull request by Neradoc**
- fix arguments in `key_number_to_row_column`.
- ~~add `on the next scan` to the description of `reset`.~~

### CODE REVIEW

File: shared-bindings/keypad/KeyMatrix.c [hunk 1]
```diff
@@ -146,8 +146,8 @@ STATIC void check_for_deinit(keypad_keymatrix_obj_t *self) {
 
 //|     def reset(self) -> None:
 //|         """Reset the internal state of the scanner to assume that all keys are now released.
-//|         Any key that is already pressed at the time of this call will therefore immediately cause
-//|         a new key-pressed event to occur.
+//|         Any key that is already pressed at the time of this call will therefore cause
+//|         a new key-pressed event to occur on the next scan.
```
Comment (dhalbert) [inline shared-bindings/keypad/KeyMatrix.c:150]: @jepler Did your recent fix make this happen immediately now?

**Review by dhalbert — COMMENTED** [review]
Thanks! One q for @jepler.

File: shared-bindings/keypad/KeyMatrix.c (hunk 1 above)
Comment (Neradoc) [reply shared-bindings/keypad/KeyMatrix.c:150]: Oh it does, I missed that !

**Comment by Neradoc** [comment]
Removed the change to the reset doc, was implemented in #6248

**Review by jepler — APPROVED** [review]
thanks

**Pull request merged**